src/main/java/ch/njol/skript/util/slot/EquipmentSlot.java

```diff
@@ -168,11 +169,11 @@ public EquipmentSlot(EntityEquipment entityEquipment, EquipSlot skriptSlot, bool
 	 * @deprecated Use {@link EquipmentSlot#EquipmentSlot(EntityEquipment, org.bukkit.inventory.EquipmentSlot)} instead
 	 */
 	@Deprecated
-	public EquipmentSlot(EntityEquipment entityEquipment, EquipSlot skriptSlot) {
+	public EquipmentSlot(@NotNull EntityEquipment entityEquipment, @NotNull EquipSlot skriptSlot) {
 		this(entityEquipment, skriptSlot, false);
 	}
 
-	public EquipmentSlot(EntityEquipment equipment, org.bukkit.inventory.EquipmentSlot bukkitSlot, boolean slotToString) {
+	public EquipmentSlot(@NotNull EntityEquipment equipment, @NotNull org.bukkit.inventory.EquipmentSlot bukkitSlot, boolean slotToString) {
 		this.entityEquipment = equipment;
```

Comment:
its probably even better if we add preconditions checks here

Reply:
Good idea